backend-rust/CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ All notable changes to this project will be documented in this file.
 
 ### Added
 
```

Comment:
Could we start having the database schema version in the changelog again, it seems that we have forgotten this in the past few releases.